service.go

```diff
@@ -22,6 +22,7 @@ type Service struct {
 	ManagedAliases             []string                     `json:"managedAliases,omitempty"`
 	Name                       string                       `json:"name,omitempty"`
 	Owner                      TeamId                       `json:"owner,omitempty"`
+	Parent                     *Identifier                  `json:"parent,omitempty" graphql:"parent"`
```

Comment:
```suggestion
        Parent                     SystemId                  `json:"parent,omitempty" graphql:"parent"`
```
This goes with my question about `GetSystem()`

Reply:
@davidbloss are you sure that we want to make this a concrete type? In GraphQL `Parent` is an optional field. 

Also, `SystemId` is just an alias for `Identifier`. Since in the codebase we tend to use `DomainId`, `SystemId`, I will change this to `SystemId`.

https://github.com/OpsLevel/opslevel-go/blob/76450060ba0eb54471934a9e2d9c946e15c71da7/system.go#L8

Reply:
Keeping this as pointer, added TODO comment about it.